engine/apps/alerts/models/alert_group_log_record.py

```diff
@@ -225,7 +233,14 @@ class AlertGroupLogRecord(models.Model):
     escalation_policy_step = models.IntegerField(null=True, default=None)
     step_specific_info = JSONField(null=True, default=None)
 
-    STEP_SPECIFIC_INFO_KEYS = ["schedule_name", "custom_button_name", "usergroup_handle", "source_integration_name"]
+    STEP_SPECIFIC_INFO_KEYS = [
+        "schedule_name",
+        "custom_button_name",
+        "usergroup_handle",
+        "source_integration_name",
+        "incident_link",
```

Comment:
could you please correct this? 🙏  In `step_specific_info` we keep `incident_id`, `incident_link` we generate later from id.
```suggestion
        "incident_id",
```